secretsmanager/src/base64.h

```diff
@@ -17,10 +17,28 @@
 
 #include <string>
 #include <vector>
+#include <memory>
+
+#include <openssl/ec.h>
+#include <openssl/evp.h>
+#include <openssl/pem.h>
 
 namespace concord::secretsmanager {
 
+/*
+ * Encode message to Base64 string.
+ * @param cipher_text (input) Message to be encoded.
+ * @return Base64 encoded message.
+ */
 std::string base64Enc(const std::vector<uint8_t>& cipher_text);
```

Comment:
rename argument to msgBytes

Reply:
Done https://github.com/arc-vmware/concord-bft/pull/6/commits/87f4edd0f4f5d72b422cef50f4a8eeeee8b28bc2